CONTRIBUTING.md

```diff
@@ -12,6 +12,19 @@ You may also check out these [other resources](https://akka.io/get-involved/).
 
 # Contributing to Alpakka
 
+## Development Setup
+
+Ensure to install the following dependencies:
+
+- [Git](https://git-scm.com/) (Make sure the `git` executable is referenced in the `PATH` environment variable)
+  - Having `git` in the `PATH` is important because it is used by the MiMa plugin when loading the sbt project.
+- Java JDK
+- [Scala 2.x](https://scala-lang.org/)
```

Comment:
Scala doesn't need to be installed. But the sbt launcher must be available.

Reply:
Thanks for that! I've Changed this line to mention SBT explicitly.